tilequeue/process.py

```diff
@@ -56,6 +56,29 @@ def _preprocess_data(feature_layers, shape_padded_bounds):
     return preproc_feature_layers
 
 
+# post-process all the layers simulataneously, which allows new
+# layers to be created from processing existing ones (e.g: for
+# computed centroids) or modifying layers based on the contents
+# of other layers (e.g: projecting attributes, deleting hidden
+# features, etc...)
+def _postprocess_data(feature_layers, post_process_data):
+
+    for step in post_process_data:
+        fn = loadClassPath(step['fn_name'])
+        params = step['params']
+
+        layer = fn(feature_layers, **params)
+        if layer is not None:
```

Comment:
Now that the intercut function won't return None, should we eliminate this check? Or do you foresee other processing functions that may want to signal failures by returning None, and we should keep this around?